packages/vite/src/node/plugins/loadFallback.ts

```diff
@@ -7,9 +7,9 @@ export function loadFallbackPlugin(): Plugin {
     name: 'load-fallback',
     async load(id) {
       try {
-        return fs.readFile(cleanUrl(id), 'utf-8')
+        return await fs.readFile(cleanUrl(id), 'utf-8')
```

Comment:
needed to do this because errors in a returned promise in `async` aren't caught by try unless the promise is awaited -- so files with # still failed even though `resolve.ts` now handles them